middleware.go

```diff
@@ -17,36 +17,30 @@
 package candlelight
 
 import (
-	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/propagation"
-	"go.opentelemetry.io/otel/trace"
 	"net/http"
 )
 
 const (
-	SpanIdHeader  = "X-B3-SpanId"
-	TraceIdHeader = "X-B3-TraceId"
+	DefaultSpanIDHeaderName  = "X-B3-SpanId"
+	DefaultTraceIDHeaderName = "X-B3-TraceId"
 )
 
-/**
-1. Acts as interceptor should be  the first point of interactions for all request's.
-2. Will  be responsible for starting a new span with existing traceId if present in request header as traceparent else it will  generate new trace id.
-	  example of traceparent will be version[2]-traceId[32]-spanId[16]-traceFlags[2] is mandatory for continuing existing traces and tracestate is optional.
-3. Will be writing the traceId and spanId in response headers for easier debugging in case of any incident.
-4. will be adding the newly created span in the request context so that we can use it other places.
-*/
-
-func TraceMiddleware(delegate http.Handler) http.Handler {
+// Acts as interceptor should be  the first point of interactions for all request's.
+// Will  be responsible for starting a new span with existing traceId if present in request header as traceparent else it will  generate new trace id.
+//	  example of traceparent will be version[2]-traceId[32]-spanId[16]-traceFlags[2] is mandatory for continuing existing traces and tracestate is optional.
+// Will be writing the traceId and spanId in response headers for easier debugging in case of any incident.
+// will be adding the newly created span in the request context so that we can use it other places.
```

Comment:
This comment doesn't follow Effective Go guidelines. Please see other comments for examples on how it can be fixed, or let me know if you want further guidance.

Reply:
Modified comments as suggested.